packages/react/src/Banner/Banner.test.tsx

```diff
@@ -115,9 +115,9 @@ describe('Banner', () => {
       />,
     )
 
-    expect(screen.getByRole('button', {name: 'test primary action'})).toBeInTheDocument()
+    expect(screen.queryAllByText('test primary action')[0]).toBeInTheDocument()
```

Comment:
is this because the tests where failing? If so maybe we should run integration tests to make sure it doesn't happen in dotcom

Reply:
For responsiveness we're rendering 2 elements now. I couldn't figure out if I could only `getByRole` for visible ones so I ended updating to get the first result

Reply:
I think the jest tests couldn't differentiate between seen and not seen 🤔 will most likely need to fix tests in dotcom when this one lands

Reply:
Is it possible to make the change in dotcom now so it doesn't break the release? We've been trying to do that if possible. If not, can we have the changes ready to go in a commit so that they can be cherry-picked into the release branch?